l10n_it_financial_statements_report/report/financial_statements_report_xlsx.py

```diff
@@ -598,7 +598,9 @@ def format_value_by_lang(self, lang, value=None, decimals=None):
         """Mimics `res.lang` model's `format` method"""
         percent = f"%.{decimals or 2}f"
         value = value or 0
-        return lang.format(percent, value, grouping=True, monetary=True)
+
+        lang = self.env["res.lang"].sudo().browse(lang.get("id"))
```

Comment:
Come mai serve usare `sudo`? Ho visto il refactoring di `res.lang` in https://github.com/odoo/odoo/commit/9f5470f247526fe54ff86e931cc835ef8942132a ma non vedo cambi di permessi di sicurezza.